localtypings/pxtarget.d.ts

```diff
@@ -779,4 +779,51 @@ declare namespace pxt.tutorial {
         // number of steps completed
         steps: number;
     }
+}
+
+declare namespace pxt.Cloud {
```

Comment:
why are we moving this in this file? This is our internal data representation and users should, in general, not take a dependency on this.

Reply:
place in a separate file.